src/UserSpaceInstrumentation/Trampoline.cpp

```diff
@@ -158,4 +160,38 @@ ErrorMessageOr<uint64_t> AllocateMemoryForTrampolines(pid_t pid, const AddressRa
   return AllocateInTracee(pid, address_range.start, size);
 }
 
+ErrorMessageOr<absl::flat_hash_set<uint64_t>> AllInstructionPointersFromProcess(pid_t pid) {
```

Comment:
nit. I'd prefer something like "GetInstructionPointersFromProcess", "l+l+I" can be hard to read.

Reply:
Or maybe GetUniqueInstructionPointersFromProcess? Is this function for test purposes only? I don't see why we would need a set, it seems to me that a hash_map would with tid as key would be more useful, no?